src/components/LoginScreen.js

```diff
@@ -14,19 +28,32 @@ export class LoginScreen extends Component {
                     </View>
                     <View style={styles.inputLayout}>
                         <TextInput 
+                            value = {this.props.login.email}
                             style = {styles.input} 
+                            onChangeText={(text) => this.props.changeEmail(text)}
                             autoCapitalize = "none"
                             placeholder="email"></TextInput>
                         <TextInput 
+                            value = {this.props.login.password}
+                            secureTextEntry={true}
                             style = {styles.input} 
+                            onChangeText={(text) => this.props.changePassword(text)}
                             placeholder="password"
                             autoCapitalize = "none"></TextInput>
                         <Button 
                             title="Login" 
-                            onPress={this.props.openProductList} />
+                            onPress={this.goToProductList} />
```

Comment:
you can directly pass the press handler, because it doesn't have arguments, and omit the creation of the goToProductList function
```
 onPress={this.props.loginAction}
```